examples/dockerfile_test.go

```diff
@@ -102,35 +103,6 @@ func TestDockerignoreSpecifiedYAML(t *testing.T) {
 	})
 }
 
-func TestDockerignoreDefaultFailYAML(t *testing.T) {
-	cwd, err := os.Getwd()
-	if !assert.NoError(t, err) {
-		t.FailNow()
-	}
-
-	integration.ProgramTest(t, &integration.ProgramTestOptions{
-		Dir:           path.Join(cwd, "test-dockerfile", "dockerignore-default-fail"),
-		Quick:         true,
-		SkipRefresh:   true,
-		ExpectFailure: true,
-	})
-}
```

Comment:
This was effectively the same as the `dockerignore-default` test. I removed it and consolidated its assertion with the other test by adding explicit existence/non-existence checks in the `Dockerfile`.